bin/kube_overrides.rb

```diff
@@ -1,7 +1,11 @@
 require 'yaml'
 require 'json'
 
-namespace, domain, kube_config = ARGV.shift(3)
+release, kube_config = ARGV.shift(3)
```

Comment:
Should be `shift(2)`, no ? (I will do the fix as part of the review)
(I noticed because I wanted to run only the new brain test, and the INCLUDE setting I provided got lost)

Reply:
Correct; sorry about that!